File: 4. Console In and Out/Problem06/QuadraticEquation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Problem6
{
    class QuadraticEquation
    {
        static void Main(string[] args)
        {
            double a = double.Parse(Console.ReadLine());
            double b = double.Parse(Console.ReadLine());
            double c = double.Parse(Console.ReadLine());

            double D = (Math.Pow(b, 2)) - 4 * a * c;
            if (D == 0)
            {
                double x = -(b) / (2 * a);
                Console.WriteLine("{0:F2}", x);
            }
            else if (D < 0)
            {
                Console.WriteLine("no real roots");
            }
            else
            {
                double x1 = (-(b) - Math.Sqrt(D)) / (2 * a);
                double x2 = (-(b) + Math.Sqrt(D)) / (2 * a);

                Console.WriteLine("{0:F2}\n{2:F2}", x1, x2);
            }
        }
    }
}

File: JavaScript-Fundamentals/05. Conditional Statements/Problem06/QuadraticEquation.js
function quadraticEquation(number) {
    var a = +number[0],
        b = +number[1],
        c = +number[2];

    var D = Math.pow(b, 2) - 4 * a * c;
    if (D == 0) {
        var x = -b / (2 * a);
        console.log('x1=x2=' + x.toFixed(2));
    } else if (D < 0) {
        console.log("no real roots");
    } else {
        var x2 = (-b - Math.sqrt(D)) / (2 * a),
            x1 = (-b + Math.sqrt(D)) / (2 * a);
        if (x1 < x2) {
            console.log('x1=' + x1.toFixed(2) + '; ' + 'x2=' + x2.toFixed(2));
        } else {
            console.log('x1=' + x2.toFixed(2) + '; ' + 'x2=' + x1.toFixed(2));
        }
    }
}